res/controllers/Midi_for_light-scripts.js

```diff
@@ -102,26 +102,30 @@ midi_for_light.init = function(id) { // called when the MIDI device is opened &
 };
 
 midi_for_light.shutdown = function(id) { // called when the MIDI device is closed
-    engine.stopTimer(midi_for_light.deck_beat_watchdog_timer[0]);
-    engine.stopTimer(midi_for_light.deck_beat_watchdog_timer[1]);
-    engine.stopTimer(midi_for_light.deck_beat_watchdog_timer[2]);
-    engine.stopTimer(midi_for_light.deck_beat_watchdog_timer[3]);
```

Comment:
Note that this likely didn't work in the first place. `deck_beat_watchdog_timer` is a free variable, not a member of `midi_for_light`. It was probably never noticed being part of the rarely invoked shutdown handler.

Reply:
The error can be reproduced by selecting the MIDI for light mapping and then switching to another mapping (which errors)